src/Components/SearchForm/SearchForm.stories.js

```diff
@@ -0,0 +1,28 @@
+import {SearchForm} from './SearchForm';
+import { userEvent, waitFor, within } from '@storybook/testing-library';
+import { expect } from '@storybook/jest';
+import '../../index.css';
```

Comment:
The name for the css file should be the same as a component name for more readability.

Reply:
Understand. But this is common css file. Haven't found how to include it for all stories by default.